src/main/java/org/broadinstitute/consent/http/service/DataAccessReportsParser.java

```diff
@@ -88,8 +88,8 @@ public void addApprovedDARLine(FileWriter darWriter, Election election, DataAcce
 
     public void addReviewedDARLine(FileWriter darWriter, Election election, DataAccessRequest dar, String darCode, String consentName, String translatedUseRestriction) throws IOException {
         String finalVote = election.getFinalVote() ? "Yes" : "No";
-        String content2 = formatTimeToDate(election.getFinalVoteDate().getTime()) + DEFAULT_SEPARATOR +
-                          finalVote;
+        String electionDate = (Objects.nonNull(election.getFinalVoteDate())) ? formatTimeToDate(election.getFinalVoteDate().getTime()) : "";
+        String content2 = electionDate + DEFAULT_SEPARATOR + finalVote;
```

Comment:
what does this variable do? maybe needs more descriptive name

Reply:
`addDARLine` has two custom content variables that it uses - this is the second one :-) Will rename.